package/gluon-web-wifi-config/luasrc/lib/gluon/config-mode/model/admin/wifi-config.lua

```diff
@@ -97,7 +108,57 @@ uci:foreach('wireless', 'wifi-device', function(config)
 	end
 end)
 
+
+if site.wifi5.outdoor_chanlist() and has_5ghz_radio() then
```

Comment:
The other two places where the outdoor mode is set - the preset and the config wizard - don't care if outdoor_chanlist is set, so this code shouldn't either. Otherwise you could upgrade from a firmware with outdoor channels to a firmware without, leaving you without an option to disable outdoor mode.

Reply:
Wrt https://github.com/freifunk-gluon/gluon/pull/1329#discussion_r278135451 the current design decision is to make the chanlist mandatory everywhere.

This was addressed in b74fc36b99b8e7a196308a432dda597b5e55e2ca (preset) and 9aed7b22fed9dfd4c93d2ff616b0d282c8ce13c9 (wizard).

Reply:
The chanlist was now made optional as we have a proper fallback channel selection through a1b317f31eb35f11c7fd32288468cdbecb9290ab.

I dropped https://github.com/freifunk-gluon/gluon/commit/b74fc36b99b8e7a196308a432dda597b5e55e2ca and https://github.com/freifunk-gluon/gluon/commit/9aed7b22fed9dfd4c93d2ff616b0d282c8ce13c9, and removed the channel list condition from web-wifi-config in e522014f44bb8d54b5f2a44e6cd409c1d880fa9f.